modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldTypeSettings.java

```diff
@@ -72,8 +75,10 @@
 								"visibilityExpression", "fieldNamespace",
 								"indexType", "labelAtStructureLevel",
 								"localizable", "nativeField", "readOnly",
-								"type", "showLabel", "repeatable", "validation",
-								"tooltip"
+								"type", "showLabel", "repeatable",
+								"requireConfirmation", "direction",
+								"confirmationLabel", "confirmationErrorMessage",
+								"validation", "tooltip"
```

Comment:
@carolmariaabb, can we sort it alphabetically or will it affect the UI behavior?

Reply:
Hi @natocesarrego, It will affect the UI. The `requireConfirmation` must come after the `repeatable` property and before the `validation` property, according to the `mockup`.

Reply:
Thanks for the clarification, @carolmariaabb! :smile: